### PR TITLE
upgraded phpstan to v0.9 and fixed level 7 issues

### PR DESCRIPTION
**Pull request by janatjak**
PR for #314

### CODE REVIEW

File: src/Kdyby/Doctrine/Diagnostics/Panel.php [hunk 1]
```diff
@@ -314,10 +314,7 @@ public function renderQueryException($e)
 			$types = $params = [];
 
 			if ($this->connection !== NULL) {
-				if (!$e instanceof Kdyby\Doctrine\DBALException || $e->connection !== $this->connection) {
-					return NULL;
```
Comment (enumag) [inline src/Kdyby/Doctrine/Diagnostics/Panel.php]: Are you sure that this can be removed?

File: src/Kdyby/Doctrine/PDOStatement.php [hunk 2]
```diff
@@ -23,9 +23,7 @@ class PDOStatement extends Statement
 {
 
 	/**
-	 * @param null $params
-	 * @throws DBALException
-	 * @return bool
+	 * @{@inheritdoc}
```
Comment (enumag) [inline src/Kdyby/Doctrine/PDOStatement.php]: The leading `@` should not be here.

File: tests/KdybyTests/Doctrine/ORMTestCase.php [hunk 3]
```diff
@@ -67,8 +67,8 @@ protected function createMemoryManager()
 
 		/** @var Nette\DI\Container $container */
 
+        /** @var Kdyby\Doctrine\EntityManager $em */
```
Comment (enumag) [inline tests/KdybyTests/Doctrine/ORMTestCase.php]: wrong indentation

File: src/Kdyby/Doctrine/Diagnostics/Panel.php (hunk 1 above)
Comment (janatjak) [reply src/Kdyby/Doctrine/Diagnostics/Panel.php]: Yes, I'm.
There is condition `$e instanceof PDOException`. It means, $e cannot be instance of Kdyby\Doctrine\DBALException.

**Comment by enumag** [comment]
I think so, squash and Ill merge it

**Comment by janatjak** [comment]
@enumag done

**Pull request merged**